main.nf

```diff
@@ -511,26 +555,27 @@ if (!params.Q2imported){
 				def folder = item.take(item.indexOf("${params.split}")) //re-extract folder
 				[ "${folder}${params.split}manifest.txt", item + '\n' ]
 			}
-			.set { ch_manifest_files }
+			.set { ch_manifest_file }
 			
-		ch_manifest_files
+		ch_manifest_file
 			.combine( ch_mpl_for_qiime_import )
 			.set { ch_manifest }
 	}
 
 	/*
 	* Import trimmed files into QIIME2 artefact
 	*/
-	if (!params.multipleSequencingRuns){
+	if (!params.multipleSequencingRuns) {
```

Comment:
Why not adding this condition also to the `when` directive inside the process?

Reply:
Because it is an if/else statement: either `process qiime_import` or `process qiime_import_multi`.

Reply:
alright, also works like this. But I find it easier to read in the code with the when statement inside the project directive. But anyways `when` will not exist any more in DSL2, so never mind!